### PR TITLE
user targeting

### PR DESCRIPTION
**Pull request by falkorichter**

### CODE REVIEW

File: _posts/2017-01-07-User Targeting.md [hunk 1]
```diff
@@ -35,9 +35,14 @@ public class MyApplication extends MultiDexApplication {
 }
 {% endhighlight %}
 
+And on iOS:
 {% highlight ObjC %}
+NSDictionary *attributes = @{ @"userLoggedIn":@YES,
```
Comment (falkorichter) [inline _posts/2017-01-07-User Targeting.md]: Android only allows String values. Can we really do this?

Comment (tagyro) [inline _posts/2017-01-07-User Targeting.md]: yes, it gets converted to a NSNumber which is perfectly safe to send

File: _posts/2017-01-07-User Targeting.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+{% highlight java %}
+public class MyApplication extends MultiDexApplication {
+	public SensorbergSdk sensorbergSDK;
+	@Override
+    public void onCreate() {
+    	//initialize your SDK properly
+    	Map<String, String> attributes = new HashMap<>();
+    	if (userMananger.userloggedIn) {
+    		attributes.put("userLoggedIn", "true");
```
Comment (budius) [inline _posts/2017-01-07-User Targeting.md]: `user.LoggedIn` (with a dot to match the next line)

File: _posts/2017-01-07-User Targeting.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    		attributes.put("user.hometown", userMananger.getHomeTown());
+    	}
+    	attributes.put("scoce", "above1000");
+    	sensorbergSDK.setAttributes(attributes);
```
Comment (budius) [inline _posts/2017-01-07-User Targeting.md]: `SensorbergSDK` (with capital letter, static method call)

**Pull request merged**